EventsExpress/ClientApp/src/components/RegistrationForm/TellUsMoreAboutYourself.js

```diff
@@ -0,0 +1,180 @@
+import relationShipStatusEnum from '../../constants/relationShipStatusEnum';
+import theTypeOfLeisureEnum from '../../constants/theTypeOfLeisureEnum';
+import reasonsForUsingTheSiteEnum from '../../constants/reasonsForUsingTheSiteEnum';
+import eventTypeEnum from '../../constants/eventTypeEnum';
+import { renderCheckbox, radioButton } from '../helpers/form-helpers';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Radio from '@material-ui/core/Radio';
+
+const TellUsMoreAbotYourself = (props) => {
+    const { handleSubmit, previousPage } = props;
+
+
+    return (
+        <div className="Step4">
+            <form onSubmit={handleSubmit}>
+                <h1 className="text-left">Step4: Tell us more about yourself. </h1>
+                <Grid
+                    container spacing={2}
+                    className="text-left">
+                    <Grid item xs={4}>
+                        <div class="stepper-tellusmore-block">
+                            <Field
+                                name="parentstatus"
+                                component={radioButton}
+                                parse={String} >
+                                <label className="font-weight-bold">Select parent status</label>
+                                <FormControlLabel
+                                    value={parentStatusEnum.Kids}
+                                    control={<Radio />}
+                                    label={parentStatusEnum.Kids} />
+                                <FormControlLabel
+                                    value={parentStatusEnum.NoKids}
+                                    control={<Radio />}
+                                    label={parentStatusEnum.NoKids} />
+                            </Field>
+                        </div>
+                        <div class="stepper-tellusmore-block">
+                            <label className="font-weight-bold">Select reasons for using the site</label>
+                            <Field
+                                name='isBeMoreActive'
+                                component={renderCheckbox}
+                                type="checkbox"
+                                label={reasonsForUsingTheSiteEnum.BeMoreActive}
+                            />
+                            <Field
+                                name='isDevelopASkill'
+                                component={renderCheckbox}
+                                type="checkbox"
+                                label={reasonsForUsingTheSiteEnum.DevelopASkill}
+                            />
+                            <Field
+                                name='isMeetPeopleLikeMe'
+                                component={renderCheckbox}
+                                type="checkbox"
+                                label={reasonsForUsingTheSiteEnum.MeetPeopleLikeMe}
+                            />
```

Comment:
I guess we can reuse [Multi-Checkbox Field renderer](https://github.com/ita-social-projects/EventsExpress/blob/development/EventsExpress/ClientApp/src/components/helpers/form-helpers/MultiCheckbox.js) here